config.md

```diff
@@ -127,4 +127,77 @@ For Linux-based systems the user structure has the following fields:
 Interpretation of the platform section of the JSON file is used to find which platform-specific sections may be available in the document.
 For example, if `os` is set to `linux`, then a JSON object conforming to the [Linux-specific schema](config-linux.md) SHOULD be found at the key `linux` in the `config.json`.
 
+## Configuration Schema Example
+
+Here is a full exmample `config.json` contents for reference (generated with `runc spec`).
```

Comment:
nit: “exmample” → “example”, and I'd drop “contents” and “(generated with `runc spec`)”.  The generation note may be useful for a commit message, but doesn't seem useful in the spec itself.